packages/core/src/domain/tracekit/index.ts

```diff
@@ -1,3 +1,3 @@
 export { Handler, StackTrace, BrowserError } from './types'
 export { computeStackTrace } from './computeStackTrace'
-export { report, subscribe, unsubscribe } from './report'
+export { subscribe, unsubscribe } from './report'
```

Comment:
`report` may not be the right name for the file anymore then.
`tracekit` instead?